source/jormungandr/jormungandr/scenarios/helper_classes/complete_pt_journey.py

```diff
@@ -50,7 +49,7 @@ def wait_and_get_pt_journeys(future_pt_journey, has_valid_direct_paths):
         if pt_journeys.error.id == response_pb2.Error.error_id.Value('no_solution') and has_valid_direct_paths:
             pt_journeys.ClearField(b"error")
         else:
-            raise PtException(pt_journeys)
+            raise PtNoSolutionException(pt_journeys)
```

Comment:
It seems strange that 'no solution' is treated as an exception. It's totally legit from the planner to return no journey ?